content/api/relay-webhooks.apib

```diff
@@ -440,6 +440,17 @@ List all your relay webhooks.
                         "name": "Replies Webhook",
                         "target": "https://webhooks.customer.example/replies",
                         "auth_token": "5ebe2294ecd0e0f08eab7690d2a6ee69",
+                        "auth_type": "oauth2",
+                        "auth_request_details": {
+                            "url": "https://oauth.myurl.com/tokens",
+                            "body": {
+                                "client_id": "<oath client id>",
+                                "client_secret": "<oath client secret>"
+                            }
+                        },
+                        "custom_headers": {
+                            "x-api-key": "abcd"
```

Comment:
Please use a different example for header name - wouldn't want customers putting their actual SparkPost api key in here. Maybe:

```
"x-webhook-source": "sparkpost"
```